### PR TITLE
Add 2015-12-18 notes

### PR DESCRIPTION
**Pull request by beaugunderson**
cc @juliepagano, @ashleygwilliams

### CODE REVIEW

**Comment by sup** [comment]
maybe do _TASK_ for better visual grepping? otherwise, looks good

**Comment by ashleygwilliams** [comment]
looks good to me! @sup's point seems positive as well. i can merge after you make that change :)

File: meetings/2015-12-18.md [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+- ashley: two weeks? we need time to be able to report the decision and what
+  happened as a result of it
+- beau/rich: seems reasonable
+- julie: sounds like the doc has a good list of types of things that happen in
+  private
+- julie: but we don't have a list of the private spaces themselves or who has
+  access to them
+- rich/ashley: that leads into the next agenda item
+- ashley: leading into membership discussion
+- rich: there's also an open PR from brian/nebrius about membership:
```
Comment (nebrius) [inline meetings/2015-12-18.md]: Nit: I spell my first name "Bryan" (don't worry, people spell it with an "i" all the time 😄)

**Comment by ashleygwilliams** [comment]
looks good to me-- want to squash? then i'll merge

**Comment by ashleygwilliams** [comment]
:tada:

**Pull request merged**